PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -3130,7 +3253,31 @@ function Get-SddcDiagnosticInfo
     # Phase 4
     #
 
-    Show-Update "<<< Phase 4 - Compacting files for transport >>>" -ForegroundColor Cyan
+    if ($ZipFiles)
+    {
+        Show-Update "<<< Phase 4 - Compacting files for transport >>>" -ForegroundColor Cyan
+    }
+    else
+    {
+        Show-Update "<<< Phase 4 - Extract cab files + Final Cleanup >>>" -ForegroundColor Cyan
+    }
+
+    if (!$ZipFiles)
+    {
+         Show-Update "Rename msdbg.<node name> to msdbg. Do this to shorten overall filepath."
+         $items = get-childitem -Recurse -Path $Path -Filter "msdbg.*"
+         foreach ($item in $items)
+         {
+             if ($item.FullName -match "msdbg(.*)")
+             {
+                 $childFolder = split-path $item.FullName -leaf

```

Comment:
minor style: core powershell cmds are normatively in Upper-CaseForm in this module. The small changes elsewhere have followed that pattern, can you switch this block to match?